addons/purchase_mrp/views/purchase_order_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo><data>
+    <record id="purchase_order_form_mrp" model="ir.ui.view">
+        <field name="name">purchase.order.inherited.form.mrp</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button class="oe_stat_button" name="action_view_mto_mrp_productions" type="object" icon="fa-wrench" attrs="{'invisible': [('mto_mrp_production_count', '=', 0)]}">
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_value"><field name="mto_mrp_production_count"/></span>
+                        <span class="o_stat_text">Manufacturings</span>
+                    </div>
+                </button>
+            </xpath>
+        </field>
+    </record>
+</data></odoo> 
```

Comment:
https://github.com/odoo/odoo/pull/43366/files/29d29286173c2f70cffd596382515aaa35bcfdf4#r367384346
The same